test/src/main/java/org/apache/accumulo/test/ComprehensiveIT_SimpleSuite.java

```diff
@@ -19,20 +19,34 @@
 package org.apache.accumulo.test;
 
 import static org.apache.accumulo.harness.AccumuloITBase.SUNNY_DAY;
+import static org.apache.accumulo.harness.SharedMiniClusterBase.getClientProps;
+import static org.apache.accumulo.test.ComprehensiveITBase.AUTHORIZATIONS;
 
 import org.apache.accumulo.core.client.Accumulo;
 import org.apache.accumulo.core.client.AccumuloClient;
+import org.apache.accumulo.core.conf.Property;
+import org.apache.accumulo.harness.MiniClusterConfigurationCallback;
 import org.apache.accumulo.harness.SharedMiniClusterBase;
+import org.apache.accumulo.miniclusterImpl.MiniAccumuloConfigImpl;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Tag;
 
 @Tag(SUNNY_DAY)
 public class ComprehensiveIT_SimpleSuite extends ComprehensiveITBase {
```

Comment:
```suggestion
public class ComprehensiveIT extends ComprehensiveITBase {
```

The setup for this test changed from starting MAC without any custom config to starting with a custom config, so it should no longer be part of the suite.